### PR TITLE
Added email.js in utils and used it in githubOauth.js, issue #170

### PR DESCRIPTION
**Pull request by udaymittal7**
issue #170 

Added the email.js file in utils. I have used nodemailer package and used gmail as service just as you asked.
You have to add EMAIL_USER and EMAIL_PASSWORD in .env file for the gmail service to use your id to send mails.
I have used this file in githubOauth.js in the config file when we are creating a new user.

Please check and let me know if any changes are needed.
Thank you.

### CODE REVIEW

File: src/utils/email.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    },
+    // activate in gmail "less secure app" option
+  });
+
+  // 1) define the email options
+  const mailOptions = {
+    from: process.env.EMAIL_USER,
+    to: mail,
+    subject: 'Welcome to CodeTrophs!',
+    text: `Hello ${this.name}.\nWe welcome you to the CodeTrophs Family.\n\nCodeTrophs is an open source project that provides an interactive user interface for beginners and aims at bridging the gap between students and research professors worldwide. Moreover, the platform is acting as a helping hand to individuals who are preparing for the coding interviews, providing them with real live interview experience.\n\nIf you have any questions, please reply to this email. We're always happy to help!\n\nCodeTrophs Team`,
```
Comment (msk4862) [inline src/utils/email.js:18]: It should be just `name` not `this.name`.

File: src/utils/email.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+const nodemailer = require('nodemailer');
+
+const email = async (name, mail) => {
+  const mailTransporter = nodemailer.createTransport({
+    service: 'gmail',
+    auth: {
```
Comment (msk4862) [inline src/utils/email.js:6]: Read environment vars in `/config/index.js` then import them here.

**Comment by msk4862** [comment]
@udaymittal7 have you done the changes?

**Comment by aryasoni98** [comment]
@udaymittal7 Any update ?

**Comment by msk4862** [comment]
@aryasoni98 I've made the required changes. So, you can merge it now. Also, you have to add email creds in `.env` file as specified in `.env.example`.
> Currently used email service is Gmail. You can also refer [this](https://nodemailer.com/usage/using-gmail/).

**Review by msk4862 — APPROVED** [review]

**Pull request merged**